### DISCUSSION TITLE
@milkdown/components/link-tooltip default labels

### DISCUSSION THREAD

**Discussion by Elgorm** [Milkdown/milkdown#1349] [Q&A] [answered]
How can I change the default labels for @milkdown/components/link-tooltip?

**Comment by Saul-Mirone** [comment, marked as answer]
```ts
import { html } from 'atomico'

ctx.update(linkTooltipConfig.key, (config) => {
  return {
    ...config,
    confirmButton: html`your own label`
  }
});
```
The default config is here: https://github.com/Milkdown/milkdown/blob/main/packages/components/src/link-tooltip/slices.ts#L41

**Reply by Elgorm** [reply]
> ```ts
> import { html } from 'atomico'
> 
> ctx.update(linkTooltipConfig.key, (config) => {
>   return {
>     ...config,
>     confirmButton: html`your own label`
>   }
> });
> ```
> 
> The default config is here: https://github.com/Milkdown/milkdown/blob/main/packages/components/src/link-tooltip/slices.ts#L41

Thanks it works. Just like this `confirmButton: () => html'your own label'`
